### ISSUE TITLE
Bug: Single click on empty space to scroll up

### ISSUE THREAD

**Issue by hishitetsu** [files-community/Files#12523]
### Description

When I single-click on empty space in the details layout, the page always scrolls up to the last selected item.

### Steps To Reproduce

1. Open the folder where scrolling occurs in the details layout.
2. Scroll down.
3. Single click on empty space.
4. The page scrolls up to the last selected item.

### Requirements

Ensure scrolling does not occur when single-clicking.

### Files Version

2.4.71.0

### Windows Version

Windows 11 22H2

### Log file

n/a

**Comment by yaira2** [comment]
Thanks for the report, this is an issue in stable as well.

**Comment by lapaz17** [comment]
@yaira2 could you assign me?

**Comment by yaira2** [comment]
Assigned

**Comment by lapaz17** [comment]
@yaira2 Is istabstop property important for details layout? Chat gpt suggested 2 solutions. And istabstop works.
````
whenever I clear selected items in listview it focuses on latest selected item. What might be the cause?
ChatGPT
If the focus automatically moves to the latest selected item when you clear the selected items in a ListView in UWP, it is likely due to the default behavior of the ListView control. By default, when you clear the selection of items in a ListView, the focus will move to the last selected item.

To prevent this behavior and remove the focus from the ListView or the selected item, you can manually set the focus to another UI element or remove the focus altogether.

Here are a couple of approaches you can try:

Set focus to another UI element:
After clearing the selected items in the ListView, you can programmatically set the focus to another UI element in your application. This can be achieved by calling the Focus() method on the desired UI element. For example, you can set the focus to a different control or the parent container of the ListView.

Remove focus from the ListView:
If you don't want the focus to move to any specific UI element after clearing the selected items, you can remove the focus from the ListView itself. One way to achieve this is by setting the IsTabStop property of the ListView to false. This prevents the ListView from receiving focus via keyboard navigation.

xaml
Copy code
<ListView IsTabStop="False" />
By setting IsTabStop to false, the ListView will no longer be able to receive focus through traditional keyboard navigation, and the focus will not move to the latest selected item when you clear the selection.

Remember to choose the approach that best suits your application's user experience and requirements.
````

**Comment by lapaz17** [comment]
No

**Comment by yaira2** [comment]
@hishitetsu this issue will be resolved with #13157
